gopdf.go

```diff
@@ -823,12 +823,36 @@ func (gp *GoPdf) SetAnchor(name string) {
 	gp.anchors[name] = anchorOption{gp.curr.IndexOfPageObj, y}
 }
 
-//AddTTFFontByReader add font file
+// ParseTTFFontByReader returns parsed TTF font data.
+func (gp *GoPdf) ParseTTFFontByReader(rd io.Reader) ([]byte, error) {
+	subsetFont := new(SubsetFontObj)
+	return subsetFont.ParseTTFByReader(rd)
+}
+
+// AddParsedTTFFont adds parsed font data which is generated by ParseTTFFontByReader.
```

Comment:
As for the naming, `AddParsedTTFFont` sounds very confusing to me because the fontdata is not something *parsed* but it's just pre-loaded original fontdata.